docs/csharp/language-reference/keywords/is.md

```diff
@@ -79,7 +79,9 @@ The `is` expression is `true` if any of the following is true:
 
 - *expr* is an instance of a type that implements the *type* interface.
 
-If *exp* is `true` and `is` is used with an `if` statement, *varname* is assigned and has local scope within the `if` statement only.
+- *expr* does not evaluate to `null`.
```

Comment:
I'll suggest moving this to Line 72, above.  Then, Line 72 would read

> The `is` expression is `true` if *expr* does not evaluate to `null`, and any of the following is true:

(By putting "*expr* does not evaluate to `null`" as one of the conditions in the list, *any* non-null object would mean `is` evaluates to `true`)